src/main/scala/chisel3/tester/internal/TestEnvInterface.scala

```diff
@@ -13,6 +18,24 @@ trait TestEnvInterface {
 
   def topFileName: Option[String]
 
+  /**
+    * Will add a TargetDirAnnotation with defaultDir with "test_run_dir" path prefix to the annotations
+    * if there is not a TargetDirAnnotation already present
+    *
+    * @param defaultDir     a default directory
+    * @param annotationSeq  annotations to add it to, unless one is already there
+    * @return
+    */
+  def addDefaultTargetDir(defaultDir: String, annotationSeq: AnnotationSeq): AnnotationSeq = {
+    if(annotationSeq.exists { case _: TargetDirAnnotation => true ; case _ => false }) {
```

Comment:
shouldn't there be a space after the if and the parens?

Reply:
Added, I have always used the no space convention. Doesn't trigger a warning from IntelliJ style checker. Convention seems mixed
```
CHICKMBP2:chick brew pr-masters > grep -R 'if(' */src | sort -u | wc
    1768   11106  201011
CHICKMBP2:chick brew pr-masters > grep -R 'if (' */src | sort -u | wc
    2297   17887  253366
```